mpd/client.go

```diff
@@ -49,6 +49,43 @@ type Client struct {
 	version string
 }
 
+// Error represents an error returned by the MPD server.
+// It contains the error number, the index of the causing command in the command list,
+// the name of the command in the command list and the error message.
+type Error struct {
+	Code             ErrorCode
+	CommandListIndex int
+	CommandName      string
+	Message          string
+}
+
+// ErrorCode is the error code of a Error.
+type ErrorCode int
+
+// ErrorCodes as defined in MPD source (https://www.musicpd.org/doc/api/html/Ack_8hxx_source.html)
+// version 0.21.
+const (
+	ErrorNotList       ErrorCode = 1
+	ErrorArg                     = 2
```

Comment:
You need to repeat the type for all consts. Otherwise, they are just ints instead of ErrorCode. See https://play.golang.org/p/qRce0yfPIOQ

Reply:
ack